cognite/powerops/utils/retry/api.py

```diff
@@ -0,0 +1,114 @@
+import logging
+import random
+import time
+
+from functools import partial, wraps
+
+
+logging_logger = logging.getLogger(__name__)
+
+
+
+def decorator(caller):
+    """ Turns caller into a decorator.
+    Unlike decorator module, function signature is not preserved.
+
+    :param caller: caller(f, *args, **kwargs)
+    """
+    def decor(f):
+        @wraps(f)
+        def wrapper(*args, **kwargs):
+            return caller(f, *args, **kwargs)
+        return wrapper
+    return decor
+
+
+def __retry_internal(f, exceptions=Exception, tries=-1, delay=0, max_delay=None, backoff=1, jitter=0,
```

Comment:
```suggestion
def _retry_internal(f, exceptions=Exception, tries=-1, delay=0, max_delay=None, backoff=1, jitter=0,
```
Single underscore to indicate private, double underscore is to indicate variable mangling in Python